src/windows/models/files_model.py

```diff
@@ -350,10 +351,10 @@ def add_files(self, files, image_seq_details=None, quiet=False, prevent_image_se
 
                 # Let the event loop run to update the status bar
                 get_app().processEvents()
+                prev_path = settings.getDefaultPath(settings.actionType.IMPORT)
 
-                prev_path = app.project.get("import_path")
                 if dir_path != prev_path:
```

Comment:
I'm not sure we need this logic anymore: `if dir_path != prev_path`... since we are only updating a setting... and no longer updating the project.